recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
+
+cron 'mysql-accounting' do
+  command '/usr/local/bin/mysql-accounting'
+  time :daily
+end
```

Comment:
ServerSpec has a [cron resource](http://serverspec.org/resource_types.html#cron) which you can use to test that the cron job has been created.

I also just pushed a fix which should allow the suites to run again. Looks like we missed importing the flavor we need to run this properly on OpenStack.